.eslint/rules/icon-has-viewBox.test.js

```diff
@@ -0,0 +1,9 @@
+// import noUnusedOwnProps from './no-stevenpetryk';
```

Comment:
right now this is just commented out code to have handy as a basic test structure, pls ignore